libs/npm/core/src/hooks/api-editor/interfaces/IChartTemplateSettingsModel.ts

```diff
@@ -1,8 +1,5 @@
 export interface IChartTemplateSettingsModel {
   chartTypes: string[];
-  isOverTime: boolean;
```

Comment:
Removed `isOverTime` for a simpler approach.  We can create separate chart templates for over time.